modules/apps/journal/journal-web/src/main/resources/META-INF/resources/item/selector/select_articles.jsp

```diff
@@ -373,8 +373,8 @@ JournalArticleItemSelectorViewDisplayContext journalArticleItemSelectorViewDispl
 	</liferay-ui:search-container>
 </clay:container-fluid>
 
-<aui:script require="metal-dom/src/all/dom as dom" sandbox="<%= true %>">
-	var selectArticleHandler = dom.delegate(
+<aui:script require="frontend-js-web/liferay/delegate/delegate.es as delegateModule" sandbox="<%= true %>">
+	var selectArticleHandler = delegate(
```

Comment:
Missing delegate variable declaration, isn't it? 
`var delegate = delegateModule.default;`

Reply:
It seems to be the case, yes @ambrinchaudhary I'll get on top of that ASAP

Reply:
@wincent what can we do to make sure the wrong version doesn't reach Brian?

Reply:
It has already been forwarded, I'll close https://github.com/brianchandotcom/liferay-portal/pull/96359 and forward it again once this is fixed ;-)

Reply:
Thank you @ambrinchaudhary

Reply:
> Missing delegate variable declaration, isn't it?

I am glad somebody is awake!